test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
> Overriding the definition would be ideal

Should we be limiting this override to top-level-defined variables or at any scopes?